File: tests/fixtures/sample.java
// Java fixture for symbol extraction tests
public class Calculator {
    private int value;

    public Calculator(int initial) {
        this.value = initial;
    }

    public void add(int amount) {
        this.value += amount;
    }

    public int getValue() {
        return this.value;
    }

    public static int multiply(int a, int b) {
        return a * b;
    }
}

class Config {
    String host;
    int port;
}

File: tests/fixtures/sample.cs
// C# fixture for symbol extraction tests
using System;

public class Calculator
{
    private int value;

    public Calculator(int initial)
    {
        this.value = initial;
    }

    public void Add(int amount)
    {
        this.value += amount;
    }

    public int GetValue()
    {
        return this.value;
    }

    public static int Multiply(int a, int b)
    {
        return a * b;
    }
}

class Config
{
    public string Host { get; set; }
    public int Port { get; set; }
}